modules/features/manatime_feature_field_bases/manatime_feature_field_bases.features.field_base.inc

```diff
@@ -81,13 +81,77 @@ function manatime_feature_field_bases_field_default_field_bases() {
     'type' => 'list_boolean',
   );
 
-  // Exported field_base: 'field_weekly_hours'.
-  $field_bases['field_weekly_hours'] = array(
+  // Exported field_base: 'field_client'.
+  $field_bases['field_client'] = array(
     'active' => 1,
     'cardinality' => 1,
     'deleted' => 0,
     'entity_types' => array(),
-    'field_name' => 'field_weekly_hours',
+    'field_name' => 'field_client',
+    'indexes' => array(
+      'target_id' => array(
+        0 => 'target_id',
+      ),
+    ),
+    'locked' => 0,
+    'module' => 'entityreference',
+    'settings' => array(
+      'handler' => 'base',
+      'handler_settings' => array(
+        'behaviors' => array(
+          'views-select-list' => array(
+            'status' => 0,
+          ),
+        ),
+        'sort' => array(
+          'type' => 'none',
+        ),
+        'target_bundles' => array(
+          'clients' => 'clients',
+        ),
+      ),
+      'target_type' => 'node',
+    ),
+    'translatable' => 0,
+    'type' => 'entityreference',
+  );
+
+  // Exported field_base: 'field_date'.
+  $field_bases['field_date'] = array(
+    'active' => 1,
```

Comment:
We need date to only contain: day, month and year. Besides that, we also need to collect end date in the same field.